configure.sh

```diff
@@ -134,6 +138,9 @@ ipo=default
 
 glpk_dir=default
 
+wasm=OFF
```

Comment:
```suggestion
wasm=default
```

To be consistent with the other options. Also, below, we are checking `"$wasm" != default`, which can only be false if we set `wasm` to `default` here.